### PR TITLE
Add singularity apps command to list installed apps in container

### LINKED ISSUE #2407: [scif-apps] Apps features are incomplete on 3.0
## Version of Singularity:

```
[eduardo@rhel7 singularity]$ singularity version
3.0.1-87.gaabae96
```
### missing features

 - [x] retrieve list of `scif-apps` on the image
how to 2.x
```
singularity apps <image.sif>
```
 - [x] Retrieve section `%apphelp` 
how to in 2.x
```
$ singularity help --app foo <image.sif>
This is the help for foo!
```
 - [ ] ability to `inspect` a specific app
how to in 2.x
```
singularity inspect --app foo <image.sif>
```

### PR DESCRIPTION
**Pull request by bauerm97**
**Description of the Pull Request (PR):**

Title

Fixes: #2407

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](../CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](../CHANGELOG.md) if necessary according to the [Contribution Guidelines](../CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](../CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](../CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: docs/content.go [hunk 1]
```diff
@@ -664,6 +664,48 @@ Enterprise Performance Computing (EPC)`
...
+  to the active app (--app <app>) and the bottom available for all 
+  apps regardless of the active app:
+
+  ACTIVE APP ENVIRONMENT:
+
+      SCIF_APPNAME       the name of the application
+      SCIF_APPROOT       the application base (/scif/apps/<app>)
+      SCIF_APPMETA       the application metadata folder
+      SCIF_APPDATA       the data base folder for active app
+        SCIF_APPINPUT    expected input folder within data base folder
```
Comment (ArangoGutierrez) [inline docs/content.go:690]: ident

**Review by ArangoGutierrez — DISMISSED** [review]
smal identation comment

File: docs/content.go (hunk 1 above)
Comment (bauerm97) [reply docs/content.go:690]: This is the appropriate indentation here, this was taken from `2.x`'s help text

**Review by ikaneshiro — APPROVED** [review]

File: cmd/internal/cli/apps_linux.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+	Args: cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		// Sanity check
+		if _, err := os.Stat(args[0]); err != nil {
+			sylog.Fatalf("Container not found: %s", err)
+		}
+
+		// apps prints the apps installed in the container
+		abspath, err := filepath.Abs(args[0])
```
Comment (tri-adam) [inline cmd/internal/cli/apps_linux.go:46]: Should check `err` here and `sylog.Fatalf` if not `nil`.

File: docs/content.go (hunk 1 above)
Comment (tri-adam) [reply docs/content.go:690]: Yeah, the indentation does look a little strange on this line.

Nitpick: the copyright header needs to be updated in this file also.

**Review by tri-adam — CHANGES_REQUESTED** [review]
Couple of small requests.

**Review by tri-adam — APPROVED** [review]

**Pull request merged**